src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj

```diff
@@ -19,5 +21,61 @@
     <Compile Remove="Console/Console.cs" />
   </ItemGroup>
 
-  <Target Name="RunSample" DependsOnTargets="RunSampleWithBrowserAndSimpleServer" />
+  <Target Name="RunSample" DependsOnTargets="$(RunSampleDependencies)" />
+
+  <Target Name="BuildWBT">
+    <PropertyGroup>
+      <_ScriptExt Condition="'$(OS)' == 'Windows_NT'">.cmd</_ScriptExt>
+      <_ScriptExt Condition="'$(OS)' != 'Windows_NT'">.sh</_ScriptExt>
+      <_Dotnet>$(RepoRoot)dotnet$(_ScriptExt)</_Dotnet>
+    </PropertyGroup>
+
+    <Exec IgnoreExitCode="true" Command="$(_Dotnet) build $(MSBuildThisFileDirectory)../../../wasm/Wasm.Build.Tests/Wasm.Build.Tests.csproj -c $(Configuration) -t:Test -p:TargetOS=browser -p:TargetArchitecture=wasm -p:XUnitClassName=none $(BuildAdditionalArgs)" />
```

Comment:
Is there a reason not to invoke target `InstallWorkloadUsingArtifacts` directly?

https://github.com/dotnet/runtime/blob/a5461cbc8ed20e0981fd4e846a180f35b07dcc0a/eng/testing/workloads-testing.targets#L149

Reply:
Good point. I changed it in the past before I found out about the `WasmOverridePacks` trick. It should work fine now, so changed it back.